docs/src/main/asciidoc/kubernetes-resources.adoc

```diff
@@ -404,14 +404,14 @@ To enable the generation of Openshift resources, you need to include Openshift i
 kubernetes.deployment.target=openshift
 ----
 
-If you need to generate resources for both platforms (vanilla kubernetes and opesnhift), then you need to include both (coma separated).
+If you need to generate resources for both platforms (vanilla kubernetes and openshift), then you need to include both (coma separated).
 
 [source]
 ----
 kubernetes.deployment.target=kubernetes, openshift
 ----
 
-The Openshfit resources can be customized in a similar approach with kubernetes.
+The Openshift resources can be customized in a similar approach with kubernetes.
```

Comment:
```suggestion
The OpenShift resources can be customized in a similar approach with kubernetes.
```

Reply:
Thank you @gsmet . Added the suggestions :)

Reply:
Looks like you forgot this one? And can you squash everything in a single commit? Thanks!

Reply:
@gsmet updated most of the case references. Please confirm :)